content/Watsonx/WatsonxGov/Level-4/202.md

```diff
@@ -0,0 +1,180 @@
+
+9. Use the **Object storage instance** dropdown to select your object storage service.
+
+10. Click **Create** to create your inventory. Inventory creation can take up to a minute to complete.
+    
+    ![](../images/202/inventory_details.png)
+
+    Once the inventory has been created, you will have the opportunity to add collaborators. Click the **x** in the upper right to close the **Set collaborators** window, then click the **x** in the upper right to close the **Manage** window.
+
+## Deploy a watsonx prompt template
+
+To deploy a watsonx model, you must first create a deployment space. A [deployment space](https://dataplatform.cloud.ibm.com/docs/content/wsj/analyze-data/ml-spaces_local.html?context=cpdaas) is an object in Cloud Pak for Data and watsonx that contains deployable assets, deployments, deployment jobs, associated input and output data, and the associated environments.
+
+> **Note**: If you are using a TechZone SaaS account, ensure that you have switched to the correct account and region in the upper right portion of the screen.
+
+   ![](../images/202/correct_account.png)
+
+1. From the watsonx home screen, click on the **hamburger menu** in the upper left.
+
+2. Click on the **Deployments** menu item. The **Deployments** screen opens.
+
+   ![](../images/202/watsonx_deployments.png)
+
+3. Click on the **New deployment space** button. The **Create a deployment space** screen opens.
+
+4. Enter `watsonx resume summarization` in the **Name** field.
+
+5. Click on the **Select storage service** dropdown and select an available Cloud Object Storage instance.
+
+6. Click on the **Select a machine learning service** dropdown and select an available Watson Machine Learning service.
+
+7. Click on the **Deployment** stage dropdown and select **Development**.
+
+> **Note**: If an object storage or machine learning service is unavailable, **DO NOT ATTEMPT TO PROVISION A NEW SERVICE**, especially if using a TechZone account. Instead, verify that you have switched to the correct account and region as specified above step 1 above. Verify that the account and region match your [TechZone reservation](https://techzone.ibm.com/my/reservations). If you are still unable to access the services, attempt to create your deployment space in your browser's private/incognito mode. If all of these steps fail, contact [TechZone help](https://techzone.ibm.com/help).
+
+   ![](../images/202/create_space.png)
+
+8. Click the **Create** button. Your deployment space will take up to one minute to create.
+
+9. When the space has been successfully created, close the **The space is ready** dialog window.
+
+   ![](../images/202/space_ready.png)
+
+Next, you will create a watsonx project containing a simple prompt template, which can be deployed to your space.
```

Comment:
Should give this section a title since it starts at step 1. If not needed, then change to step 10 and update the following step numbers as well